docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,49 +1,54 @@
 _Please request review from `@zooniverse/frontend` team or an individual member of that team._ 
 
-Package:
-
-Closes # .
+## Package:
 
+## Linked Issue and/or Talk Post:
 
 ## Describe your changes:
 
-<em>Helpful explanations that will make your reviewer happy:</em>
+## How to Review
+_Helpful explanations that will make your reviewer happy:_
 - What Zooniverse project should my reviewer use to review UX?
 - What user actions should my reviewer step through to review this PR?
 - Which storybook stories should be reviewed?
 - Are there plans for follow up PR’s to further fix this bug or develop this feature?
 
-
-# Review Checklist
+# Checklist
+_PR Creator - Please cater the checklist to fit the review needed for your code changes._
+_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._
```

Comment:
My goal here is to encourage the PR creator to step through the relevant checklist items themself _before_ pushing a new branch and requesting review. I think syncing the PR creator and PR reviewer's understanding of the PR's changes will be beneficial to our team's limited time resources.